app/services/sync_service/events.rb

```diff
@@ -65,7 +66,7 @@ def record_hash(event)
   def create_or_update_if_needed!(record_hash)
     # If a record already exists with this content hash, then no update needed
     if should_create_or_update(record_hash)
-      event = Event.find_by(content_hash: record_hash["content_hash"]) || Event.find_by(guid: record_hash["guid"])
+      event = Event.find_by(content_hash: record_hash["content_hash"]) || Event.find_by(guid: record_hash["guid"], timestamp_start: record_hash["timestamp_start"])
```

Comment:
@cdoyle-temple Using this logic, if someone changes the time of a non-repeating event for typical reasons, like they made a typo or they are rescheduling it for the following day, that will create a new event instance, so there will be vents for both the original time and the new time